deployments/properties/azure-functions-linux-dotnet-rbac.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy .NET Core app to Azure Functions App on Linux using RBAC",
+    "description": "Build a .NET Core project and deploy it to an Azure Functions App on Linux using Azure role-based access control (Azure RBAC).",
+    "creator": "Microsoft Azure",
+    "iconName": "azure",
+    "categories": ["Deployment", "C#", "aspNetCore"]
```

Comment:
Do we expect the repos to have either `host.json` or `function.json` file. If yes, then you should add "[Azure Functions](https://github.com/github-starter-workflows/repo-analysis-partner/blob/0b80be121c78ca30896c6cbdb209b89cc5f3d0e0/tech_stacks.yml#LL20C3-L20C18)" as one of the categories.

Reply:
Added Azure Functions to all